payload/payload.cc

```diff
@@ -52,6 +53,7 @@ void createInitialGlobalState(unique_ptr<core::GlobalState> &gs, const realmain:
         auto workers = WorkerPool::create(emptyOpts.threads, gs->tracer());
         auto indexed = realmain::pipeline::index(gs, payloadFiles, emptyOpts, *workers, kvstore);
         realmain::pipeline::resolve(gs, move(indexed), emptyOpts, *workers); // result is thrown away
+        gs->ensureCleanStrings = false;
```

Comment:
this only checks it during initial global state construction, right?
I don't think this covers usecases that we want checked. We want to run Sorbet test suite on inputs from test suite with it enabled.

Reply:
@DarkDimius: Yes it only checks payload loading.

> We want to run Sorbet test suite on inputs from test suite with it enabled.

That's the part I didn't understood, now I see the problem, sorry!